### PR TITLE
[Bug]: fixed typo ob_dml_stmt_printer.cpp roolback

### PR DESCRIPTION
**Pull request by SaumyaBhushan**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=bHLE6ZQ)

### What changes were proposed in this pull request?
[Bug]: fixed typo ob_dml_stmt_printer.cpp roolback

### Why are the changes needed?
There was a typo which needed to be fixed

### Will break the compatibility? How if so?
No

### Does this PR introduce any user-facing change?

No
### How was this patch tested?

https://github.com/oceanbase/oceanbase/issues/1215
### Checklist

- [ ] I've run the tests to see all new and existing tests pass.
- [x] If this Pull Request resolves an issue, I linked to the issue in the text above.
- [x] I've informed the technical writer about the documentation change if necessary.

### CODE REVIEW

**Review by hnwyllmm — APPROVED** [review]

**Comment by SaumyaBhushan** [comment]
@hnwyllmm One workflow is still waiting for approval .

**Comment by hnwyllmm** [comment]
Don't worry. It can't block the progress发自我的 iPhone在 2022年12月3日，01:13，Saumya ***@***.***> 写道：﻿
@hnwyllmm One workflow is still waiting for approval .

**Review by hnwyllmm — APPROVED** [review]

**Pull request merged**
